drivers/display/display_st7735r.c

```diff
@@ -268,7 +269,7 @@ static void st7735r_get_capabilities(const struct device *dev,
 	memset(capabilities, 0, sizeof(struct display_capabilities));
 	capabilities->x_resolution = config->width;
 	capabilities->y_resolution = config->height;
-	if (config->madctl & ST7735R_MADCTL_BGR) {
+	if (!(config->madctl & ST7735R_MADCTL_BGR) != !config->rgb_is_inverted) {
```

Comment:
```suggestion
	if ((config->madctl & ST7735R_MADCTL_BGR) && !config->rgb_is_inverted) {
```

Reply:
Under the condition, the result is as follows.
It has bit problem.

|  MADCTL   |  inverted   |  result   |
|----|----|---|
|RGB| not inverted | RGB (false)|
|RGB| inverted | *RGB (false)* |
|BGR| not inverted | BGR (true)|
|BGR| inverted | RGB (false)|

The original expression is essentially XOR.

Reply:
@jfischer-no 
Thanks for comment.
I was not careful enough. I will be careful next time.

Reply:
I added a comment because it was difficult to understand and unkindly.